src/main/kotlin/com/terraformation/backend/seedbank/db/AccessionStore.kt

```diff
@@ -867,11 +868,21 @@ class AccessionStore(
                     1)
                 .else_(0))
 
+    val speciesQuery =
+        dslContext
+            .select(DSL.countDistinct(ACCESSIONS.SPECIES_ID))
+            .from(ACCESSIONS)
+            .innerJoin(SPECIES)
+            .on(ACCESSIONS.SPECIES_ID.eq(SPECIES.ID))
+            .and(SPECIES.DELETED_TIME.isNull)
```

Comment:
The first `.and` should be a `.where` here. The end result will be the same in this case because it's a simple inner join, but semantically, the accession ID and species deletion status aren't determining how to join `accessions` and `species` together.